src/Configurations/ListPane/List.js

```diff
@@ -0,0 +1,55 @@
+import React, { useMemo } from 'react';
+import { FormattedMessage, useIntl } from 'react-intl';
+import moment from 'moment';
```

Comment:
I'm trying to not use moment in new code these days given [the Moment project's suggestion that you probably shouldn't be using it anymore](https://momentjs.com/docs/#/-project-status/). At a glance it seems like the vanilla JS libs could satisfy your requirements here - is that the case or did I not dig enough?

Reply:
That's from the code before. I just left it as-is.
What do we use in folio instead?

Reply:
Ahhh, hard to tell when it just shows up as a new file in Github! In general, vanilla JS' `Date` API is good enough for everything I've needed in Folio, beforehand. That link in the comment has some alternatives, including libraries if vanilla isn't good enough.

Reply:
I guess, I'll leave it a tech debt for now.
Too many changes for one PR )